scripts/cp-pkgs.js

```diff
@@ -77,9 +90,24 @@ function cpAsset(asset) {
   return cpFile(asset, destDir).then(() => console.log(`cp ${asset} -> ${destDir}`));
 }
 
+function cpDeclarationAsset(asset) {
+  const assetPkg = path.parse(asset.split('build/')[1]).dir;
+  if (!fs.existsSync(assetPkg)) {
+    Promise.reject(new Error(`Non-existent asset package path ${assetPkg} for ${asset}`));
+  }
+  const destFileName = toCamelCase(path.parse(asset).name.replace(/^mdc-|\.d$/gm, ''));
```

Comment:
Why is the `/m` ([multiline](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#Syntax)) switch needed on this RegExp?

Reply:
removed